### DISCUSSION TITLE
posts/goodbye_spotify

### DISCUSSION THREAD

**Discussion by giscus** [raed667/blog-comments#2] [Announcements]
# posts/goodbye_spotify

I used to be a big fan of Spotify. Today I cancelled my subscription.

http://localhost:3000/posts/goodbye_spotify

**Comment by gramster** [comment]
While it’s certainly not without its problems, I support artists I like by still buying their music on Bandcamp (or Beatport). And some even have Patreons. Those companies still take too much of a cut but it’s way better than Spotify.

**Comment by lolski** [comment]
Your reason for stopping to use Spotify is the same as mine. I want software to be un-opinionated building blocks. Let me have control to customise what i want to see. Let me have control over the content. If i decided to add a song X, that means i want to own it. From the time I added it and all the way to decades later until and if I decide to delete it.

**Comment by agneevX** [comment]
I'm in the same boat. I moved away from Spotify because of the same reasons but I found solace in Apple Music even though recommendations suck, it's slow and the desktop apps suck.

**Comment by hergaiety** [comment]
All of these streaming platforms are struggling in somewhat similar ways with Spotify and Netflix being giant examples of how _not_ to do things. Appreciate this writeup and experiential callout.

This year I decided to see how much music I already had on Bandcamp and, funny enough, Humble Bundle to start up my own playlists of songs I like which inspired me to ditch streaming platforms as well. It's been nice to fall back in love with music by caring about my music collection while being able to pay artists semi-directly.

**Comment by tpnguyen88** [comment]
Basic enshittification of all platforms today. Streaming is a trap, go back to mp3.

**Comment by fourstepper** [comment]
What are you using instead?

**Comment by lolski** [comment]
For me I stick with collecting my own mp3s. The player I'm using is this one https://brushedtype.co/doppler/

**Comment by wehrend** [comment]
I have to add, there is a reason why spotify emphases the podcasts, because they are bound to the decisions of the big three players, and they also decide where the money is going, so it is not really spotify to be blamed here.
See this video: https://www.youtube.com/watch?v=szxizxMm1ww

**Comment by sam-k3nny** [comment]
Nice story. I have all my music ripped to mp3s which I save to a Synology NAS. I use https://www.plex.tv/plexamp/ to stream them on my iPhone. It’s been a real joy rediscovering music I own - CDs I purchased years ago. I rarely use Spotify any more.

**Comment by leonewt0n** [comment]
I switch between iTunes and Youtube music + adblock. I mainly just use youtube music now, and apple podcasts.

Thats cool you literally have the same blog template as me! I was just thinking about adding comments to it and giscus seems like what I need!

**Reply by raed667** [reply]
I've tried Youtube music + adblock, but I feel that the recommendation engine is very repetitive and somehow always in the same order.

Astro is really cool for blogging ! i've added a couple of very minor tweaks to the theme, but it has been overall great and the comments were easy to add
